### PR TITLE
EZP-29099: Implement permissions for Section management in Admin Panel

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29099
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

For now this PR implement permissions policy 'Section/Assign' without UDW. 

screen shot 2018-05-10 at 10 53 11 am

screen shot 2018-05-10 at 10 53 41 am

screen shot 2018-05-10 at 10 52 59 am

screen shot 2018-05-10 at 10 52 32 am

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [ ] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/admin/section/view.html.twig [hunk 1]
```diff
@@ -25,20 +25,22 @@
...
+                    {% if deletable %}
+                        <button type="button" class="btn btn-danger" data-toggle="modal"
+                                data-target="#delete-section-modal" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
+                            <svg class="ez-icon">
+                                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
+                            </svg>
+                        </button>
+                        {% include 'EzPlatformAdminUiBundle:admin/section:delete_confirmation_modal.html.twig' with {'form': form_section_delete} %}
+                    {% else %}
+                        <button type="button" class="btn btn-danger disabled" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
```
Comment (dew326) [inline src/bundle/Resources/views/admin/section/view.html.twig]: I think we should not repeat this code, it's the same button no matter if we can delete or not. The disabled state on the button should be added when is not deletable.

File: src/bundle/Resources/views/admin/section/list.html.twig [hunk 2]
```diff
@@ -35,19 +35,19 @@
                             <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#create"></use>
                         </svg>
                     </a>
+                    {% set modal_data_target = 'delete-sections-modal' %}
+                    <button id="delete-sections" type="button" class="btn btn-danger" disabled data-toggle="modal"
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/section/list.html.twig]: In order to improve code readability, please move each attribute onto a separate line.

Comment (mikadamczyk) [reply src/bundle/Resources/views/admin/section/list.html.twig]: @sunpietro done

File: src/bundle/Resources/views/admin/section/list.html.twig [hunk 3]
```diff
@@ -35,19 +35,24 @@
                             <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#create"></use>
                         </svg>
                     </a>
+                    {% set modal_data_target = 'delete-sections-modal' %}
+                    <button
+                        id="delete-sections"
+                        type="button"
+                        class="btn btn-danger"
+                        disabled data-toggle="modal"
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/section/list.html.twig]: `disabled` should go on to a separate line

**Pull request merged**